packages/postgres/src/pool.ts

```diff
@@ -924,7 +927,23 @@ export class PostgresPool {
           client.release();
         }
       },
+      stream: function (batchSize: number) {
+        const transform = new Transform({
+          readableObjectMode: true,
+          writableObjectMode: true,
+          transform(chunk, _encoding, callback) {
+            for (const row of chunk) {
+              this.push(row);
+            }
+            callback();
+          },
+        });
+
+        // TODO: use native `node:stream#compose` once it's stable.
```

Comment:
It's currently listed as experimental and isn't present in the TypeScript types for Node, so I opted to avoid it for now.